aiida/orm/implementation/django/node.py

```diff
@@ -488,9 +488,9 @@ def store_all(self, with_transaction=True):
                 "Node with pk= {} was already stored".format(self.pk))
 
         # For each parent, check that all its inputs are stored
-        for label in self._inputlinks_cache:
+        for link in self._inputlinks_cache.itervalues():
```

Comment:
yes it is a LinkInfo you are right, but again since we are working now on reworking the whole link system anyway, I won't bother